generate.py

```diff
@@ -55,23 +55,32 @@ def logits_to_probs(logits, temperature: float = 1.0, top_k: Optional[int] = Non
     return probs
 
 
-def sample(logits, temperature: float = 1.0, top_k: Optional[int] = None):
-    probs = logits_to_probs(logits[0, -1], temperature, top_k)
+def sample(logits, temperature: float = 1.0, top_k: Optional[int] = None, ignore_gist: bool = True):
+    next_logits = logits[0, -1]
+    if ignore_gist:
```

Comment:
I would add a comment stating why this is necessary and/or add a "TODO - figure out why model generating it"